<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorGroupWidget</class>
 <widget class="QWidget" name="InspectorGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>132</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutNoteField">
     <item>
      <spacer name="horizontalSpacerNotes">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>33</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelNotes">
       <property name="text">
        <string>Notes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEditNotes">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAutoStep">
     <item>
      <spacer name="horizontalSpacerAutoStep1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelAutoStep">
       <property name="text">
        <string>Auto step</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoStep">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerAutoStep2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAutoPlay">
     <item>
      <spacer name="horizontalSpacerAutoPlay1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>8</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelAutoPlay">
       <property name="text">
        <string>Auto play</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoPlay">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerAutoPlay2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerGroup">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>labelNotes</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorGroupWidget</receiver>
   <slot>resetNotes(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEditNotes</sender>
   <signal>textChanged()</signal>
   <receiver>InspectorGroupWidget</receiver>
   <slot>notesChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelAutoStep</sender>
   <signal>linkActivated(QString)</signal>
   <receiver>InspectorGroupWidget</receiver>
   <slot>resetAutoStep(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoStep</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorGroupWidget</receiver>
   <slot>autoStepChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoPlay</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorGroupWidget</receiver>
   <slot>autoPlayChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetNotes(QString)</slot>
  <slot>notesChanged()</slot>
  <slot>resetAutoStep(QString)</slot>
  <slot>autoStepChanged(int)</slot>
  <slot>autoPlayChanged(int)</slot>
 </slots>
</ui>
